diff -up /media/1/tmp/rineau-BUILD/Manual_tools/install.sh.rpm /media/1/tmp/rineau-BUILD/Manual_tools/install.sh
--- /media/1/tmp/rineau-BUILD/Manual_tools/install.sh.rpm	2007-02-20 23:12:23.000000000 +0100
+++ install.sh	2008-09-18 16:22:33.000000000 +0200
@@ -2,6 +2,8 @@
 
   . ./install.config
 
+make -C src LATEX_CONV_INPUTS=$LATEX_CONV_INPUTS || exit 1
+
 install -d $LATEX_CONV_BIN $LATEX_CONV_CONFIG $STYLE_FILES $STYLE_FILES/eps_tabs $STYLE_FILES/eps_tabs_grey
 
 cd src
