--- scripts/cc_ref_wizard~	2006-02-14 11:08:15.000000000 +0100
+++ scripts/cc_ref_wizard	2006-08-17 13:28:12.000000000 +0200
@@ -21,7 +21,7 @@
 # Path to the configuration files of the latex_converter: It contains
 # the skeletons for the reference manual pages that will be costumized
 # by this skript.
-# set LATEX_CONV_CONFIG = /CGAL/Tools/latex_converter_config
+set LATEX_CONV_CONFIG = /usr/share/CGAL/latex_conv_config
 
 # Author of the refernce page
 # set REF_WIZARD_AUTHOR = ""
