Index: install.sh
===================================================================
--- install.sh	(revision 29111)
+++ install.sh	(working copy)
@@ -2,14 +2,18 @@
 
 source install.config
 
+make -C src LATEX_CONV_INPUTS=$LATEX_CONV_INPUTS || exit 1
+
+LATEX_CONV_BIN=${RPM_BUILD_ROOT}${LATEX_CONV_BIN}
+LATEX_CONV_CONFIG=${RPM_BUILD_ROOT}${LATEX_CONV_CONFIG}
+STYLE_FILES=${RPM_BUILD_ROOT}${STYLE_FILES}
+
 install -d $LATEX_CONV_BIN $LATEX_CONV_CONFIG $STYLE_FILES $STYLE_FILES/eps_tabs $STYLE_FILES/eps_tabs_grey
 
-cd src
-#make clean
-make || exit 1
-make install
-cd ..
 
+
+make -C src install LATEX_CONV_CONFIG=$LATEX_CONV_CONFIG|| exit 1
+
 cp scripts/index_fix scripts/cc_make_ref_pages scripts/cc_ref_wizard $LATEX_CONV_BIN
 cp sty/*.sty $STYLE_FILES
 cp sty/eps_tabs/*.pdf sty/eps_tabs/*.eps $STYLE_FILES/eps_tabs
@@ -18,4 +22,4 @@
 echo ""
 echo "================================================"
 echo "Manual_tools successfullly installed. Do not forget to update \$TEXINPUTS:"
-echo "export TEXINPUTS=\".:$STYLE_FILES:\$TEXINPUTS\""
\ No newline at end of file
+echo "export TEXINPUTS=\".:$STYLE_FILES:\$TEXINPUTS\""
